### DISCUSSION TITLE
getting_started/introduction/learning_new_features

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#68] [User-contributed notes]
# getting_started/introduction/learning_new_features

Godot is a feature-rich game engine. There is a lot to learn about it. This page explains how you can use the online manual, built-in code reference, and join online communities to learn new featur...

https://docs.godotengine.org/en/stable/getting_started/introduction/learning_new_features.html

**Comment by Falco-Boehnke** [comment]
For Windows I would suggest "Snipping Tool" for taking screenshots. It's natively installed on basically all Versions of Windows now. Can be found in the startmenu when you type the name in.
It allows for precise "cutting" of the screen and some basic functionality to highlight areas.

**Reply by MichaelMerrett** [reply]
Also, Windows+Shift+S is the keybind to open it.

**Reply by MightyPossum** [reply]
Or windows+shift+R to record a video of a selection

**Comment by Lainelle** [comment]
>or FlameShot on Linux.

On Windows too, I use it ^^

**Reply by Calinou** [reply]
Pull requesst opened:

- https://github.com/godotengine/godot-docs/pull/11055

**Comment by Natlaan** [comment]
In at least one of the other documentation pages, GD-Quest was mentioned as a recommendation for beginners. Apart from that, this useful tip doesn't seem to be included here or anywhere else.

**Reply by Calinou** [reply]
Learn GDScript from Zero is linked on the [Tutorials](https://docs.godotengine.org/en/stable/community/tutorials.html#where-to-start) page linked at the bottom of this page.
